src/styles/_catalog.scss

```
@@ -1,22 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-auto-rows: 300px;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

```suggestion
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }

  @media (min-width: 768px) {
    grid-template-columns: repeat(3, 200px);
  }

  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, 200px);
  }
```

Припускаю, що ось тут можна створити цикл, щоб не дублювати код

---

+1

---

Really good approach to avoid duplication of code  without loss of  readability. Done.